.devcontainer/Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN set -eux; \
     mesa-libGLU-devel \
     ncurses-devel \
     ninja-build \
+    python-devel \
```

Comment:
I approve but why is this necessary?

Reply:
Its in the commit message. Boost Python fails to build without it. I thought this was in my pull request but realized later that I overlooked committing and pushing.